data/src/main/scala/ch.epfl.scala.index.data/project/ProjectConvert.scala

```diff
@@ -28,7 +28,7 @@ class ProjectConvert(paths: DataPaths, githubDownload: GithubDownload)
    */
   def convertAll(
       pomsRepoSha: Iterable[(ReleaseModel, LocalRepository, String)],
-      cachedReleases: Map[Project.Reference, Set[Release]]
+      cachedReleases: Map[Project.Reference, Seq[Release]]
```

Comment:
If I try to re-publish to scaladex a release that was already indexed, are going to count it twice?

Reply:
```suggestion
      indexedReleases: Map[Project.Reference, Seq[Release]]
```

Or, `oldReleases`, but you already use this identifier below.

Reply:
> If I try to re-publish to scaladex a release that was already indexed, are we going to count it twice?
No because there is filter in the `IndexingActor`

indexedReleases is a better name indeed :)

Reply:
I plan to split this method in two because it has two distinct usage:
- converting a newly published artifact to a project and a release
- converting all the artifacts that are stored in the filesystem and group them by project

The current `Map[Project.Reference, Seq[Release]]` signature has no sense in any of the two usages.